ajax/libs/wangEditor/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "wangEditor",
+  "version": "2.1.23",
+  "filename": "js/wangEditor.min.js",
+  "description": "wangEditor - 基于javascript和css开发的 Web富文本编辑器 轻量简洁易用开源免费(wangEditor based on javascript and css that makes it lightweight, easy to create simple, free and opensource.)",
+  "homepage": "https://wangeditor.github.io/",
```

Comment:
The `https://wangeditor.github.io/` or `http://wangeditor.github.io/` will redirect to `http://www.wangeditor.com/`
`github.io` domain name support ssl, maybe we can use `https`.
What do you think?